js/tabs/static/Tabs.svelte

```diff
@@ -58,11 +59,15 @@
 	<div class="tab-nav scroll-hide">
 		{#each tabs as t, i (t.id)}
 			{#if t.id === $selected_tab}
-				<button class="selected">
+				<button
+					class="selected"
+					{...elem_id ? { id: t.elem_id + "-button" } : {}}
```

Comment:
In Svelte if you set an attribute to null it gets removed. That might be a slightly cleaner way of expressing this.

Reply:
Ah nice. Figured there'd a nicer way. Will update